### DISCUSSION TITLE
Suporte para Livewire 3

### DISCUSSION THREAD

**Discussion by buenojosias** [wireui/wireui#695] [General]
Este é sem dúvida o melhor pacote UI para Livewire. Eu o uso em todos os meus projetos.
Mas eu gostaria de saber se ele terá suporte para LW 3, pois pretendo atualizar alguns dos projetos logo que ele for lançado.

**Comment by josepueblaphcs** [comment]
I want to know the same thing. I love Wire UI but I'm so hype for the new features on Livewire 3 that I want to upgrade.

**Comment by kennyelliott** [comment]
I'm also interested in Livewire 3 support. I've got a project in the process of being built and we want to move to Livewire 3.

**Comment by insulae** [comment]
v1 and v2 (the next version) will be support livewire 3. 
you can read here in the roadmap:
https://github.com/wireui/wireui/issues/502
